### ISSUE TITLE
project is not set for google_compute_zones

### ISSUE THREAD

**Issue by Yanson** [GoogleCloudPlatform/terraform-google-managed-instance-group#20]
Error:
`data.google_compute_zones.available: project: required field is not set`

Fix:
Project should be set here:
https://github.com/GoogleCloudPlatform/terraform-google-managed-instance-group/blob/master/main.tf#L124

E.g.:
```ruby
data "google_compute_zones" "available" {
  region = "${var.region}"
  project = "${var.project}"
}
```

**Comment by Yanson** [comment]
FYI
```
$ terraform -version
Terraform v0.11.7
+ provider.google v1.16.0
```

**Comment by danisla** [comment]
Thanks for reporting, and for fixing! Fixed with #21
